templates/default/slurm.conf.erb

```diff
@@ -27,7 +27,7 @@ SwitchType=switch/none
 MpiDefault=none
 SlurmctldPidFile=/var/run/slurmctld.pid
 SlurmdPidFile=/var/run/slurmd.pid
-ProctrackType=proctrack/pgid
+ProctrackType=proctrack/cgroup
```

Comment:
not sure how much it helps but we could have a kitchen test verifying at least that cgroups are available. you could run `cat /proc/mounts` and grep for cgroups?

also support for cgroups becomes a new requirement for customami creation. Are we tracking this anywhere in our docs?

Reply:
if within the requirements there is a package to be installed, we must add it to the list of package we install in the AMI

Reply:
Don't think we can do `cat /proc/mounts` and grep for cgroups. `alinux1` and `centos6` originally come with no `cgroup` mount points, `slurmd` actually mounts the `cgroup/freezer` directory for these 2 OSs.

For kitchen tests on compute we are not really starting `slurm`, so I would not expect `cgroup/freezer` to be there.

I agree that we need to add to our documentation to make sure users are aware that their kernel need to be compatible with `cgroup`, but I think the the majority of the OSs out there should already be compatible.
`cgroup` should come with any OS using kernel version after Linux 2.6.24, which was released 24 January 2008.